source/UniverseObjects.cpp

```diff
@@ -334,9 +353,36 @@ void UniverseObjects::LoadFile(const string &path, bool debugMode)
 	if(debugMode)
 		Logger::LogError("Parsing: " + path);
 
+	bool active = true;
```

Comment:
We should probably print a warning if the end of the file is reached and active is false, i.e. we encountered an ?IF with no ?ELSE or ?ENDIF.